integration.js

```diff
@@ -11,62 +11,72 @@ function doLookup(entities, { host, ..._options }, callback) {
   };
```

Comment:
I'd try to reduce the use of the spread operator here unless you really need it as it's not a free operation and increases garbage collection.  In this case you're destructing into `_option` and then destructing immediately again into `integrationOptions`.  The integration here is getting it's own "personal" copy of the user options so mutating the options object shouldn't be an issue unless you need to hang onto the original option formatting.